vault/logical_system_paths.go

```diff
@@ -1455,8 +1715,29 @@ func (b *SystemBackend) metricsPath() *framework.Path {
 				Query:       true,
 			},
 		},
-		Callbacks: map[logical.Operation]framework.OperationFunc{
-			logical.ReadOperation: b.handleMetrics,
+		Operations: map[logical.Operation]framework.OperationHandler{
+			logical.ReadOperation: &framework.PathOperation{
+				Callback: b.handleMetrics,
+				Responses: map[int][]framework.Response{
+					http.StatusOK: {{
+						Description: "OK",
+						Fields: map[string]*framework.FieldSchema{
+							logical.HTTPContentType: {
+								Type:     framework.TypeString,
+								Required: true,
+							},
+							logical.HTTPRawBody: {
+								Type:     framework.TypeString,
+								Required: true,
+							},
+							logical.HTTPStatusCode: {
+								Type:     framework.TypeInt,
+								Required: true,
+							},
```

Comment:
this is the old way for how vault did responses, but I dont know what the fields of the actual response would be

Reply:
cc @averche